sdk/src/ingredient.rs

```diff
@@ -705,13 +708,7 @@ impl Ingredient {
     }
 
     fn thumbnail_from_assertion(assertion: &Assertion) -> (String, Vec<u8>) {
-        let thumbnail_format = extension_to_mime(
-            &get_thumbnail_image_type(&assertion.label_root()).unwrap_or("".into()),
-        );
-        (
-            thumbnail_format.unwrap_or("image/none").to_string(),
-            assertion.data().to_vec(),
-        )
+        (assertion.content_type(), assertion.data().to_vec())
```

Comment:
Do we want this data even if no thumbnail is added before/later? Will the call be skipped if there is no thumbnail data?